models/players.js

```diff
@@ -568,6 +570,24 @@ var insertTopScorers= function (data){
   
 }
 
+var abbreviationHelper = function(league, abbreviation){
```

Comment:
i didnt realize there were 2, so i moved the function over to the players model and referenced it there